src/devices/tuya.ts

```diff
@@ -912,6 +912,8 @@ const definitions: Definition[] = [
             tuya.whitelabel('Nous', 'P3Z', 'Smart light bulb', ['_TZ3210_cieijuw1']),
             tuya.whitelabel('Moes', 'ZLD-RCW_1', 'RGB+CCT Zigbee LED controller', ['_TZ3000_7hcgjxpc']),
             tuya.whitelabel('Moes', 'ZLD-RCW_2', 'RGB+CCT LED bulb', ['_TZ3210_rcggc0ys']),
+            tuya.whitelabel('Moes', 'xxxxx', 'RGB+CCT Smart Downlight', ['_TZ3210_s9lumfhn']),
```

Comment:
Can you use the real model number (listed on the device/product page) instead of `xxxx` here?

Reply:
Hey @Koenkk ,

There was nothing on the box, not om the label at the back of the fixture, nothing on AliExpresss where I bought those...

I have found something on the big interweb: [Moes House](https://moeshouse.com/products/zigbee-smart-light-zb-lzd7-rcw-en?variant=39331508125777), I found a SKU for the 10W, but also for different power consumptions
- ZB-LZD5-RCW-EN
- ZB-LZD7-RCW-MS
- ZB-LZD10-RCW-EN
- ZB-LZD15-RCW-EN

I only have the 10W version, but are you feeling safe to use `ZB-LZD-RCW` four all four variants ? Hoping all four use the same manufacturer  `_TZ3210_s9lumfhn`
Or do you expect different manufacturers for the different lamps ?

I can go for safe, and make it:
```
            tuya.whitelabel('Moes', 'ZB-LZD10-RCW-EN', '10W RGB+CCT Smart Downlight', ['_TZ3210_s9lumfhn']),
```

Please, any advice or recommendations are appreciated 

Theo

Reply:
I have update the file, so you can merge it if you think that is the right wat to deal with the minor problem I described above.
I can easily update the MR again